src/app/garfile/arrival/post.controller.js

```diff
@@ -116,8 +107,15 @@ module.exports = async (req, res) => {
 
   // Define voyage
   const voyage = req.body;
-  voyage.arrivalPort = _.toUpper(voyage.arrivalPort);
   delete voyage.buttonClicked;
+  if (voyage.portChoice === 'No') {
```

Comment:
Looks at the choice made and sets values as appropriate.
To clear the coordinates, empty strings are used, as it appears undefined or null get ignored when sent to the back end.